packages/conf-liblz4/conf-liblz4.1/opam

```diff
@@ -11,6 +11,7 @@ depexts: [
   ["lz4-devel"] {os-distribution = "centos"}
   ["lz4-devel"] {os-distribution = "rhel"}
   ["lz4-devel"] {os-distribution = "fedora"}
+  ["lz4-devel"] {os-distribution = "oraclelinux-8"}
```

Comment:
Does this work? I think the distribution is "ol"

Reply:
It didn't. I tried `oraclelinux` and `oraclelinux-8` before. Thanks for the pointer for `ol`, I'll give that one a try.

Sidenote: how do I learn about the possible names? Especially if I am not running that distribution. I can often get package names from online search interfaces but the OS names/distribution names are difficult to figure out.

Reply:
Kate added them here: https://github.com/ocaml/opam-repository/wiki/Depexts-os-distribution---os-family-values
I used to search among the conf-package in the opam-repository before :D